tensorbay/opendataset/BDD100K/loader.py

```diff
@@ -315,48 +315,54 @@ def _mots_loader(path: str) -> Dataset:
 def _load_mots_segment(dataset: Dataset, root_path: str) -> None:
     images_directory = os.path.join(root_path, "images", "seg_track_20")
     labels_directory = os.path.join(root_path, "labels", "seg_track_20", "polygons")
-    unlabeled_segment = dataset.create_segment("unlabeled")
-    for segment_name in _SEGMENT_NAMES:
-        segment = dataset.create_segment(segment_name)
+    for segment_prefix in _SEGMENT_NAMES:
```

Comment:
`_SEGMENT_NAMES` -> `_SEGMENT_PREFIXES`?

Reply:
It's reused in other functions as segment_name